src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -0,0 +1,133 @@
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""``verdi presto`` command."""
+
+from __future__ import annotations
+
+import pathlib
+import re
+import typing as t
+
+import click
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+from aiida.cmdline.utils import echo
+from aiida.manage.configuration import get_config_option
+
+DEFAULT_PROFILE_NAME_PREFIX: str = 'presto'
+
+
+def default_profile_name():
```

Comment:
When looking at the tests I got confused by the generic name of this function. Would suggest to be more explicit.

```suggestion
def default_presto_profile_name():
```